src/unix/haiku/mod.rs

```diff
@@ -352,6 +352,66 @@ s_no_extra_traits! {
         __unused1: *mut ::c_void, // actually a function pointer
         pub sigev_notify_attributes: *mut ::pthread_attr_t,
     }
+
+    #[cfg(target_arch = "x86_64")]
+    pub struct fpu_state {
+        pub control: ::c_ushort,
+        pub status: ::c_ushort,
+        pub tag: ::c_ushort,
+        pub opcode: ::c_ushort,
+        pub rip: ::c_ulong,
+        pub rdp: ::c_ulong,
+        pub mxcsr: ::c_uint,
+        pub mxcsr_mask: ::c_uint,
+        _fpreg: [::c_uchar; 16],
+        _xmm: [[::c_uchar; 16]; 16],
+        _reserved_416_511: [[::c_uchar; 3]; 32],
```

Comment:
Any reason why you are making me do math of 3*32 instead of just setting it to 96 bytes like in the header?